ex4/ex4.c

```diff
@@ -10,7 +10,26 @@
 
 int main(void)
 {
-    // Your code here    
+    int child = fork();
+
+    if (child < 0)
+    {
+        printf("\nFork failed\n");
+    }
+    else if (child == 0)
+    {
+        char *args[2];
+        args[0] = "/bin/ls"; // list files and dirs
```

Comment:
I like how you commented here, definitely more comments are helpful as pseudocode, keep it up!